jax_rocm_plugin/jax_plugins/rocm/plugin_setup.py

```diff
@@ -26,6 +26,7 @@
 default_rocm_path = "/opt/rocm"
 rocm_path = os.getenv("ROCM_PATH", default_rocm_path)
 rocm_detected_version = rocm_path.split("-")[-1] if "-" in rocm_path else "unknown"
+rocm_tag = os.getenv("ROCM_VERSION_EXTRA")
```

Comment:
Sorry, I might be pedantic on my side but I don't see this as an improvement. Now if `ROCM_VERSION_EXTRA` is not set rocm_tag is `""` and the version will include just `+rocm` without the ROCm version which I don't think it is the intended behavior. What we need here is a fallback mechanism that is guaranteed to give us the ROCm version with which we are building the artifact when the environment variable is not set. Maybe using `rocminfo`?